### DISCUSSION TITLE
How do I clear just the scrollback buffer?

### DISCUSSION THREAD

**Discussion by salmankhilji** [kovidgoyal/kitty#7771] [General]
I want to clear the scrollback buffer—nothing more, nothing less.

When I run the following command in my terminal, it gives me exactly the behavior I want:

```
printf '\e[3J'
```

The scrollback is cleared, and my prompt stays in place.

However, when I add the following to my `kitty.conf`:

```
map f1 clear_terminal scrollback active
map f2 send_text all \e[3J
```

Neither F1 nor F2 produces the desired outcome.

- F1 erases the screen and leaves me without a PS1 prompt.
- F2 just beeps.

See: #6460

**Comment by kovidgoyal** [comment]
You need to configure your shell to send that escape code. There are
examples of how to do that in the kitty docs:
https://sw.kovidgoyal.net/kitty/conf/#shortcut-kitty.Reset-the-terminal
